### PR TITLE
dae: update to 0.9.0

### PR DESCRIPTION
**Pull request by Integral-Tech**

### CODE REVIEW

File: net/dae/Makefile [hunk 1]
```diff
@@ -5,12 +5,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dae
-PKG_VERSION:=0.8.0
+PKG_VERSION:=0.9.0
```
Comment (1715173329) [inline net/dae/Makefile:8]: add `GO_PKG_EXCLUDES:=control/kern/tests` below `GO_PKG` to disable broken tests.

Comment (1715173329) [reply net/dae/Makefile:8]: and `-DMAX_MATCH_SET_LEN=64` should be changed to `-DMAX_MATCH_SET_LEN=1024` in favor of upstream update

**Comment by Integral-Tech** [comment]
@1715173329 Done :)

**Comment by 1715173329** [comment]
Merged, thank you!

**Pull request merged**